devlib/utils/android.py

```diff
@@ -195,13 +195,16 @@ def __init__(self, device=None, timeout=None):
         AdbConnection.active_connections[self.device] += 1
         self._setup_ls()
 
-    def push(self, source, dest, timeout=None):
+    def is_rooted(self):
+        pass
```

Comment:
In the end, I removed the need for the connections to be able to say whether or not they are rooted. It was called in the target (is_rooted property) and as gem5 is always rooted, I added this 'shortcut', which is actually not needed.